data/markers.json

```
@@ -28621,5 +28621,68 @@
 		"x": 33954,
 		"y": 32076,
 		"z": 15
+	},
+	{
+		"description": "Water Buffalo spawn",
+		"icon": "star",
+		"x": 32820,
+		"y": 32168,
+		"z": 7
+	},
+	{
+		"description": "Water Buffalo spawn",
+		"icon": "star",
+		"x": 32863,
+		"y": 32141,
+		"z": 7
+	},
+	{
+		"description": "Water Buffalo spawn",
+		"icon": "star",
+		"x": 32876,
+		"y": 32199,
+		"z": 7
+	},
+	{
+		"description": "Water Buffalo spawn",
+		"icon": "star",
+		"x": 32893,
+		"y": 32236,
+		"z": 7
+	},
+	{
+		"description": "Water Buffalo spawn",
+		"icon": "star",
+		"x": 32948,
+		"y": 32260,
+		"z": 7
+	},
+	{
+		"description": "Water Buffalo spawn",
+		"icon": "star",
+		"x": 32962,
+		"y": 32223,
+		"z": 7
+	},
+	{
+		"description": "Water Buffalo spawn",
+		"icon": "star",
+		"x": 33060,
+		"y": 32143,
+		"z": 7
+	},
+	{
+		"description": "Water Buffalo spawn",
+		"icon": "star",
+		"x": 33093,
+		"y": 32076,
+		"z": 7
+	},
+	{
+		"description": "Dwarf Mines",
+		"icon": "down",
+		"x": 32427,
+		"y": 31988,
+		"z": 10
```

Should this be https://tibiamaps.io/map#32428,31989,10:3?

```suggestion
		"x": 32428,
		"y": 31989,
		"z": 10
```